types/paypal__checkout-server-sdk/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for @paypal/checkout-server-sdk 1.0
+// Project: https://github.com/paypal/Checkout-NodeJS-SDK
+// Definitions by: Martin Badin <https://github.com/martin-badin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from './lib/lib';
+export * from './lib/common-definitions';
```

Comment:
this creates more than is in the original:
https://github.com/paypal/Checkout-NodeJS-SDK
we should avoid situation like `webpack` 4 files proliferation.
can this be avoided? (re-export from lib.d.ts? there is no 'lib/common-definitions'
aren't those being used only in a single file? `types/paypal__checkout-server-sdk/lib/orders/lib.d.ts`

Reply:
I understand what you mean. It can be used in lib.d.ts. Common-definition.d.ts includes common types for both groups orders and payments. Therefor I don't know which is the correct place to move those types. If you mean the orders is a good place to move I can do it.

Reply:
done